sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -39,6 +39,13 @@ import (
 // Example 2: 777777777777.dkr.ecr.cn-north-1.amazonaws.com.cn/my_image:latest
 var ecrRegex = regexp.MustCompile(`(^[a-zA-Z0-9][a-zA-Z0-9-_]*)\.dkr\.ecr\.([a-zA-Z0-9][a-zA-Z0-9-_]*)\.amazonaws\.com(\.cn)?.*`)
 
+// derived from: https://github.com/containerd/containerd/blob/f5e7fe0cb6286333a597a407029fe7c760626669/pkg/labels/validate.go#L25-L28
+const (
+	maxSize = 4096
+	// maximum length of key portion of error message if len of key + len of value > maxSize
+	keyMaxLen = 64
+)
+
```

Comment:
Not so much derived as copy/pasted, which is generally not what you want to do as it can trigger licensing changes.

Here it would be better to give the constant we need a more specific name:
```suggestion
const (
	imageLabelMaxSize = 4096
)
```

I don't think `keyMaxLen` is necessary; we aren't expecting `host-ctr` to be invoked with very long keys.